utils/graphql.js

```diff
@@ -35,6 +35,93 @@ function logErrorsFromDS(apiResponse) {
   }
 }
 
+const request = async (query, variables) => {
+  console.debug('GraphQL query sent:', query, variables);
+
+  try {
+    return await graphQLClient.request(query, variables);
+  } catch (err) {
+    console.error('API has returned error', err);
+    logErrorsFromDS(err);
+    // eslint-disable-next-line no-throw-literal
+    throw 'Error from DS API';
+  }
+};
+
+const getSimplePsyInfo = (cursor, state) => {
+  const query = gql`
+    {
+      demarche (number: ${config.demarchesSimplifieesId}) {
+        dossiers (state: ${state}${cursor ? `, after: "${cursor}"` : ''}) {
+          pageInfo {
+            hasNextPage
+            endCursor
+          }
+          nodes {
+              id
+              state
+              datePassageEnInstruction
+              champs (id: "Q2hhbXAtMTY1OTUxMw==") {
```

Comment:
We can put this id in the configuration. If it differs from DS test and prod spaces, it would be easier to manage.